ticketing/model.go

```diff
@@ -1,19 +1,41 @@
-package main
-
-//this is an arbitrary pick
+package ticketing
+
+// EventSlot holds information for any sellable/giftable slot we have in the event for
+// a Talk or any other activity that requires admission.
+type EventSlot struct {
+	ID          uint64
+	Name        string
+	Description string
+	Cost        int64
+	Capacity    int // int should be enough even if we organize glastonbury
+	StartDate   uint64
+	EndDate     uint64
+	// DependsOn means that these two Slots need to be acquired together, user must either buy
+	// both Slots or pre-own one of the one it depends on.
+	DependsOn *EventSlot
+	// PurchaseableFrom indicates when this item is on sale, for instance early bird tickets are the first
+	// ones to go on sale.
```

Comment:
add the timestamp format to the comment?  data is a timestamp stored as milliseconds since...  apply to the rest of the date fields stored this way too.